atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresQueryFactory.java

```diff
@@ -44,7 +47,7 @@ public FullQuery getLatestRowQuery(byte[] row,
         String query =
                 " /* SQL_MET_GET_LATEST_ROW_INNER (" + tableName + ") */ " +
```

Comment:
All of these SQL_MET s are still hanging around... they we're modified in other classes.